src/main/steps/applicant2/how-to-apply-financial-order/template.njk

```diff
@@ -0,0 +1,8 @@
+{% extends "common/page.njk" %}
+
+{% block page_content %}
```

Comment:
dont think we need to repeat it if its the same. just include applicant 1s template